ui/lobby/src/view/realTime/chart.ts

```diff
@@ -31,8 +31,9 @@ const clockX = (dur: number) => {
 };
 
 function renderPlot(ctrl: LobbyController, hook: Hook) {
-  const bottom = Math.max(0, ratingY(hook.rating) - 2),
-    left = Math.max(0, clockX(hook.t) - 2),
+  const isMobilePhone = window.screen.width <= 480;
+  const bottom = Math.max(0, ratingY(hook.rating) - (isMobilePhone ? 3 : 2)),
```

Comment:
it's about computing the rating Y, and the clock X, so I think it should happen in the corresponding functions.

Also this looks very crude and hack-ish. Is there a better way to gracefully handle all screen size? Can we figure out what causes the offset instead of patching it this way?

Reply:
@ornicar It seems the issue was that on phones, the ratio of the challenge icon font size vs the chart dimensions is higher than on a PC. A bigger icon (relative to the chart) causes the icon to appear slightly higher. So I've updated the PR to calculate adjust amounts based on these factors.